sdk/tests/tutorials/instruments/test_complex_instruments.py

```diff
@@ -106,3 +106,31 @@ def test_create_fx_option(self):
         self.assertEqual(
             saved_fx_option.is_delivery_not_cash, fx_option.is_delivery_not_cash
         )
+
+        # to be inserted here
+    def test_create_term_deposit(self):
+
+        term_deposit = lusid.TermDeposit(
+            startDate=datetime(2020, 2, 5, 00, tzinfo=pytz.utc),
+            maturityDate=datetime(2020, 2, 8, 00, tzinfo=pytz.utc),
+            contractSize=1000000,
+            flowConvention=lusid.FlowConventions(
+                scope=None,
+                code=None,
+                currency="GBP",
+                paymentFrequency="6M",
+                rollConvention="MF",
+                dayCountConvention="Act365",
+                paymentCalendars=[],
+                resetCalendars=[],
+                settleDays=1,
+                resetDays=0
+                ),
+            rate=0.03,
+            instrument_type="TermDeposit"
+        )
+
+        # Assert instrument was created
+        self.assertIsNotNone(term_deposit)
+
```

Comment:
Ideally there would be a further set of assertions under this test, as is the case in other tests above and in the C# example.

Reply:
Usually I would agree more assertions the better.
Looks like in this example, we'll keep the assertions light, as these operations have already been tested in core LUSID code:
https://github.com/finbourne/lusid-sdk-python-preview/pull/88#issuecomment-802686799
(FYI - @rizwansaeed )